internal/transform/parquet_converter.go

```diff
@@ -0,0 +1,414 @@
+// This file includes interfaces and functions for converting data structures/schemas
+// to the appropriate parquet data structure/schema.
+
+package transform
+
+import (
+	"encoding/json"
+)
+
+type SchemaParquet interface {
+	ToParquet() interface{}
+}
+
+func toJSONString(v interface{}) string {
+	jsonData, err := json.Marshal(v)
+	if err != nil {
+		return ""
+	}
+	return string(jsonData)
+}
+
+func (lo LedgerOutput) ToParquet() interface{} {
```

Comment:
Also what was the decision between converting to `int64` versus keeping as `int32`?

Reply:
Yup will add a comment.

I needed to convert uint32 -> int64 otherwise uint32 could lose data `uint32 max=4,294,967,295` vs `int32 max=2,147,483,647`

Note I did convert uint64 to int64 and if that becomes a problem we would need to look for a resolution (larger data type, custom bigint as a string)